pkg/loki/loki.go

```diff
@@ -372,6 +372,8 @@ type Loki struct {
 	deleteClientMetrics *deletion.DeleteRequestClientMetrics
 
 	HTTPAuthMiddleware middleware.Interface
+
+	ingesterRelease *atomic.Bool
```

Comment:
Sorry for the late review but is this variable set anywhere?

Reply:
lol looks like something I was using in an older implementation but forgot to clean, thanks for catching it

Reply:
yep I was using it in a former implementation